docs/overrides/home.html

```diff
@@ -634,7 +642,7 @@ <h1 class="text-2xl font-bold tracking-tight text-black sm:text-4xl">Fast, Decla
 </div>
 
 <div class="overflow-hidden bg-white py-14">
-  <div class="mx-auto max-w-5xl md:px-6 lg:px-8">
+  <div class="mx-auto max-w-7xl md:px-6 lg:px-8">
     <div class="grid grid-cols-1 gap-x-8 gap-y-16 sm:gap-y-20 lg:grid-cols-2 lg:items-start">
       <div class="px-4 lg:px-0">
         <div class="relative isolate overflow-hidden rounded-md sm:mx-auto sm:max-w-2xl lg:mx-0 lg:max-w-none">
```

Comment:
Not directly related to this PR, but I thought I'd comment here since you're already fixing a bunch of stuff so seems easier than making an issue/forking.  

The code block title for this section should be `devenv.yaml`, not `.nix`.